.ado/release.yml

```diff
@@ -1,212 +1,269 @@
-    artifact: Wheels.Linux.${{ variables['arch'] }}
-    displayName: Upload Python Artifacts Linux
-    condition: eq(variables['Agent.OS'], 'Linux')
-
-- job: "Merge"
-  dependsOn:
-  - Build
-  pool:
-    vmImage: 'ubuntu-latest'
-
-  steps:
-  - download: current
-    artifact: Wheels.Win.x86_64
-    displayName: Download x86_64 Python Artifacts Win
-
-  - download: current
-    artifact: Wheels.Mac.x86_64
-    displayName: Download Python Artifacts Mac
-
-  - download: current
-    artifact: Wheels.Mac.aarch64
-    displayName: Download Python Artifacts Mac
-
-  - download: current
-    artifact: Wheels.Mac.universal
-    displayName: Download Python Artifacts Mac
-
-  - download: current
-    artifact: Wheels.Linux.x86_64
-    displayName: Download x86_64 Python Artifacts Linux
-
-  - script: |
-      mkdir -p                            target/wheels
-      mv ../Wheels.Linux.x86_64/*.whl     target/wheels
-      mv ../Wheels.Win.x86_64/*.whl       target/wheels
-      mv ../Wheels.Mac.x86_64/*.whl       target/wheels
-      mv ../Wheels.Mac.aarch64/*.whl      target/wheels
-      mv ../Wheels.Mac.universal/*.whl    target/wheels
-      ls                                  target/wheels/*
-    displayName: Move Py Artifacts to Publishing Dir
-
-  - publish: $(System.DefaultWorkingDirectory)/target/wheels
-    artifact: wheels
-    displayName: Upload Python Artifacts
-
-
-- job: "Approval"
-  dependsOn:
-  - Build
-  - Merge
-
-  pool: server
-  timeoutInMinutes: 1440 # job times out in 1 day
-  steps:
-  - task: ManualValidation@0
-    timeoutInMinutes: 1440 # task times out in 1 day
-    inputs:
-      notifyUsers: ''
-      instructions: 'Please verify artifacts and approve the release'
-      onTimeout: 'reject'
-
-
-- job: "Publish_Python_Packages"
-  dependsOn: Approval
-  pool:
-    vmImage: 'ubuntu-latest'
-
-  steps:
-  - download: current
-    artifact: wheels
-    displayName: Download Python Artifacts
-
-  - script: |
-      mkdir -p                            target/wheels
-      mv ../wheels/*.whl                  target/wheels
-      ls                                  target/wheels/*
-    displayName: Move Py Artifacts to Publishing Dir
-
-# Add ESRP steps here
-
-  - task: EsrpRelease@4
-    displayName: Publish Py Packages
-    inputs:
-     ConnectedServiceName: 'ESRP_Release'
-     Intent: 'PackageDistribution'
-     ContentType: 'PyPi'
-     FolderLocation: '$(System.DefaultWorkingDirectory)/target/wheels'
-     Owners: '$(OwnerPersonalAlias)@microsoft.com'  # NB: Group email here fails the task with non-actionable output.
-     Approvers: 'billti@microsoft.com'
-     # Auto-inserted Debugging defaults:
-     ServiceEndpointUrl: 'https://api.esrp.microsoft.com'
-     MainPublisher: 'QuantumDevelpmentKit'          # ESRP Team's Correction (including the critical typo "Develpm").
-     DomainTenantId: '72f988bf-86f1-41af-91ab-2d7cd011db47'
+  LLVM_VERSION: 14
+
+resources:
```

Comment:
Being this is a public repo in a different org, we should maybe have some comments explaining what these 1ES templates are.